content/artifacts/artifactory/plugin.js

```diff
@@ -364,9 +395,25 @@ function Untaken({ selected }) {
   });
 
   return html`
+  <div>
```

Comment:
Does this require the outer wrapper? I think preact allows multiple root nodes, but have't tried it myself.